src/parser/jobs/rdm/modules/Gauge.tsx

```diff
@@ -0,0 +1,307 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import JOBS from 'data/JOBS'
+import {Event, Events} from 'event'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Data} from 'parser/core/modules/Data'
+import {CounterGauge, Gauge as CoreGauge} from 'parser/core/modules/Gauge'
+import {Statistics} from 'parser/core/modules/Statistics'
+import Suggestions, {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import {DualStatistic} from 'parser/jobs/rdm/statistics/DualStatistic'
+import React, {Fragment} from 'react'
+import {isSuccessfulHit} from 'utilities'
+
+interface GaugeModifier {
+	white: number
+	black: number
+}
+
+export class Gauge extends CoreGauge {
+	static override title = t('rdm.gauge.title')`Mana Gauge Usage`
+
+	@dependency private data!: Data
+	@dependency private suggestions!: Suggestions
+	@dependency private statistics!: Statistics
+
+	private _whiteManaGauge = this.add(new CounterGauge({
+		graph: {
+			label: <Trans id="rdm.gauge.resource.whitemana">White Mana</Trans>,
+			color: JOBS.WHITE_MAGE.colour,
+		},
+		maximum: 100,
+		minimum: 0,
+	}))
+	private _blackManaGauge = this.add(new CounterGauge({
+		graph: {
+			label: <Trans id="rdm.gauge.resource.blackmana">Black Mana</Trans>,
+			color: JOBS.BLACK_MAGE.colour,
+		},
+		maximum: 100,
+		minimum: 0,
+	}))
+	public _gaugeModifiers = new Map<number, GaugeModifier>([
+		[this.data.actions.VERAERO.id, {white: 11, black: 0}],
+		[this.data.actions.VERAERO_II.id, {white: 7, black: 0}],
+		[this.data.actions.VERSTONE.id, {white: 9, black: 0}],
+		[this.data.actions.VERHOLY.id, {white: 21, black: 0}],
+		[this.data.actions.VERTHUNDER.id, {white: 0, black: 11}],
+		[this.data.actions.VERTHUNDER_II.id, {white: 0, black: 7}],
+		[this.data.actions.VERFIRE.id, {white: 0, black: 9}],
+		[this.data.actions.VERFLARE.id, {white: 0, black: 21}],
+		[this.data.actions.JOLT.id, {white: 3, black: 3}],
+		[this.data.actions.JOLT_II.id, {white: 3, black: 3}],
+		[this.data.actions.IMPACT.id, {white: 3, black: 3}],
+		[this.data.actions.SCORCH.id, {white: 7, black: 7}],
+	])
+	public _spenderModifiers = new Map<number, GaugeModifier>([
+		[this.data.actions.ENCHANTED_REPRISE.id, {white: -5, black: -5}],
+		[this.data.actions.ENCHANTED_MOULINET.id, {white: -20, black: -20}],
+		[this.data.actions.ENCHANTED_RIPOSTE.id, {white: -30, black: -30}],
+		[this.data.actions.ENCHANTED_ZWERCHHAU.id, {white: -25, black: -25}],
+		[this.data.actions.ENCHANTED_REDOUBLEMENT.id, {white: -25, black: -25}],
+	])
+	private _severityWastedMana = {
+		1: SEVERITY.MINOR,
+		20: SEVERITY.MEDIUM,
+		80: SEVERITY.MAJOR,
+	}
+	private _severityLostMana = {
+		1: SEVERITY.MINOR,
+		20: SEVERITY.MEDIUM,
+		80: SEVERITY.MAJOR,
+	}
+	public readonly _manaDifferenceThreshold = 30
+	private readonly _manaLostDivisor = 2
+	private readonly _manaficationMultiplier = 2
+	public readonly _manaCap = 100
```

Comment:
These would be preferred as non-exported `const` values outside of the analyser class definition (typically right after the import declarations).  Use the `const MANA_DIFFERENCE_THRESHOLD` naming convention.

Reply:
updated with suggestion.